<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetFormatDialog</class>
 <widget class="QDialog" name="DataSetFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="BrowseFilePath" name="widget_outpath" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="unify_categories">
     <property name="text">
      <string>unify categories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="debug_mode">
     <property name="text">
      <string>debug mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Train Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="SliderAndSpinBoxWidget" name="slider_train" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="SliderAndSpinBoxWidget" name="slider_minival" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="image_w">
        <property name="text">
         <string>COCO_width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="image_size_w">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image_h">
        <property name="text">
         <string>COCO_height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="image_size_h">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="color_b">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="color_g">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="color_r">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>mask</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QSpinBox" name="mask_separation_linewidth">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="cityscapes_hierarchy_format">
        <property name="text">
         <string>contours + hierarchy format for cityscape</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>separation linewidth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="generate_voc">
        <property name="text">
         <string>VOC</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="generate_coco">
        <property name="text">
         <string>COCO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="generate_cityscapes">
        <property name="text">
         <string>Cityscapes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="start_format_dataset">
     <property name="text">
      <string>Start Format DataSet</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BrowseFilePath</class>
   <extends>QWidget</extends>
   <header>ui/BrowseFilePath.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliderAndSpinBoxWidget</class>
   <extends>QWidget</extends>
   <header>ui/SliderAndSpinBoxWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
